test/unit/queue/src/QueueTest.cpp

```diff
@@ -76,15 +76,18 @@ TEMPLATE_LIST_TEST_CASE(
     Fixture f;
 
     std::atomic<bool> callbackFinished{false};
+    std::atomic<bool> inKernelIsNotEmptyBeforeSleep{false};
+    std::atomic<bool> inKernelIsNotEmptyAfterSleep{false};
+
     alpaka::enqueue(
         f.m_queue,
-        [&f, &callbackFinished]() noexcept
+        [&f, &callbackFinished, &inKernelIsNotEmptyBeforeSleep, &inKernelIsNotEmptyAfterSleep]() noexcept
```

Comment:
```suggestion
        [&]() noexcept
```
I think I would just capture all now. Applies to the other lambdas as well.

Reply:
I also thought about it but since we where explicit before I kept it.

Reply:
done